spec/spec_helper.rb

```diff
@@ -20,7 +20,7 @@
 
   # SimpleCov.refuse_coverage_drop is only implemented for line coverage, so for
   # branch coverage we must use `minimum_coverage`
-  SimpleCov.minimum_coverage branch: 90
+  SimpleCov.minimum_coverage branch: 100
 
   SimpleCov.refuse_coverage_drop
```

Comment:
I don't have strong feelings, but it might be more explicit if we changed this to also be `100`. Happy either way though!

Reply:
Going to leave as is for now!

I have [a PR up on SimpleCov](https://github.com/simplecov-ruby/simplecov/pull/934) to hopefully get branch coverage working for `:refuse_coverage_drop`, so hoping to just use refuse_coverage_drop for both eventually.